pkg/management/postgres/restore.go

```diff
@@ -209,6 +210,14 @@ func (info InitInfo) restoreDataDir(backup *apiv1.Backup, env []string) error {
 		return err
 	}
 	log.Info("Restore completed")
+
+	filePath := filepath.Join(info.PgData, archiver.CheckEmptyWalArchiveFile)
+	// We create the check empty wal archive file to tell that we should check if the
+	// destination path it is empty
+	if err := fileutils.CreateEmptyFile(filepath.Clean(filePath)); err != nil {
```

Comment:
I think ConfigureNewInstance is invoked also inside the restore process, so maybe this is not needed here

Reply:
fixed